docs/_tutorials/messagevpn-with-queue-java.md

```diff
@@ -4,55 +4,56 @@ title: Message VPN with Queue – Java
-
-This tutorial will guide you through the steps of creating a sample Java-based tool to administer basic message-VPNs and queues.
+* Demonstrate how to integrate the management of Solace messaging routers into Java-based DevOps tools using the SEMP Java Client Library
 
 Let's review the admin objectives:
 
-* **Creating message-VPNs** is a convenient way to slice a Solace message router to separated independent virtual routers, multiplying its use. There are many use cases including security, controlled share of functionality and capacity. For developers, one way to make use of a message-VPN is to share a VMR with everyone having a dedicated message-VPN for development use. Once a message-VPN is created, the client can connect to it with the assigned username and password using basic internal authentication.
+* **Creating message-VPNs** is a convenient way to slice a Solace messaging router to separated independent virtual routers, multiplying its use. There are many use cases including security, controlled share of functionality and capacity. For developers, one way to make use of a message-VPN is to share a VMR with everyone having a dedicated message-VPN for development use. Once a message-VPN is created, the client can connect to it with the assigned username and password using basic internal authentication.
 * Another common objective is to **create a persistent queue** as a message endpoint in the particular message-VPN. This resource is only visible to those connected to this message-VPN and it can store incoming guaranteed messages until consumed. 
 * The administrator may also want the ability to **delete a queue** or 
 * **delete a message-VPN** when no longer needed.
 
-To accomplish this, we will create a set of tasks as building blocks that match the SEMP object model of the Solace message router and use a simple framework to demonstrate the integration and orchestration of these tasks to achieve above objectives. We will use the SEMP Java Client Library to implement these tasks, extending the concepts introduced in the [Basic Operations - Java Client Library]({{ site.baseurl }}/java-client) tutorial.
+To accomplish this, we will create a set of tasks as building blocks that match the SEMP object model of the Solace messaging router and use a simple framework to demonstrate the integration and orchestration of these tasks to achieve above objectives. We will use the SEMP Java Client Library to implement these tasks, extending the concepts introduced in the [Basic Operations - Java Client Library]({{ site.baseurl }}/java-client) tutorial.
 
 ### Assumptions
 
 This tutorial assumes that you have access to a running Solace VMR with the following configuration:
 
-* A management user `admin` with password `admin`, authorized with a minimum access scope level of *global/read-write*.
+* A management user authorized with a minimum access scope level of *global/read-write*.
```

Comment:
Solace cloud management users only have 'message-vpn/read-write'. So likely this can work with more scoped permissions. This was an error in the original tutorial.